### PR TITLE
Use MediaRemote framework to get the album artwork

### PR DESCRIPTION
**Pull request by Trsvsr**
This PR aims to provide a different method of getting the album artwork through MediaRemote instead of relying on the iTunes API which can often return bad matches. This new function will get the Base64-encoded artwork data from the now playing info dictionary and upload it to FreeImage.host with their public API key then use that URL as the large image URL for the rich presence. I've also implemented an on-disk artwork cache (located at `~/Library/Application Support/MusicRPC/artworkCache.plist`) consisting of the Base64-encoded data as well as the matching URL to prevent unnecessary uploads in cases such as listening to a song multiple times as well as multiple songs sharing the same artwork. Feel free to make any suggestions if something looks off; I'm sure this could be cleaned up a bit.

### CODE REVIEW

**Comment by AlexandraAurora** [comment]
I appreciate the effort, however I prefer to not have the app upload something anywhere. I'm also currently implementing my own caching, while improving localization and refactoring the code.

Aside from that, this pull request adds multiple features, which should be split into two separate pull requests. The commits also aren't signed.

TLDR: Thanks for the effort, but I won't merge it.
